es/ios/crash-reporting.mdown

```diff
@@ -0,0 +1,145 @@
+
+```objc
+- (void)crash {
+    [NSException raise:NSGenericException format:@"Everything is ok. This is just a test crash."];
+}
+      ```
+```swift
+NSException(name: NSGenericException, reason: "Everything is ok. This is just a test crash.", userInfo: nil).raise()
+```
+
+Then, put this invocation in your main view controllers' `viewDidLoad`, or in `application:didFinishLaunchingWithOptions:` if you're using the App Delegate:
+
+```objc
+[self performSelector:@selector(crash) withObject:nil afterDelay:5.0];
+```
+```swift
+dispatch_after(
+    dispatch_time(DISPATCH_TIME_NOW, Int64(5.0 * Double(NSEC_PER_SEC))),
+    dispatch_get_main_queue(),
+    { () -> Void in
+        self.crash()
+});
+```
+
+A continuación, sigue estos pasos:
+
+1.  Compila, corre, y para la ejecucion del app.
+2.  Corre el app sin el debuger, el cual capturara los errores y prevendra que sean enviados a Parse. La manera mas facil de hacer esto es hacerlo directamenete en el lanzamiendo del app desde la pantalla de home.
+3.  Espera que la app tenga error.
+4.  Corre el app de nuevo. El app automaticamente enviara el repoder de error a Parse en el lanzamiento.
+
+En este punto, ve hacia el dashboard del Reporte de Errores debajo de Analytics para tu app y asegurate de ver el error en el listado. Puede tomar unos minutos para mostrarse.
+
+## Flujo de trabajo
+
+Usando los Reportes de Crash, puedes incrementar dramaticamente la calidad de tu app, reduciendo el numero de errores que tus usuarios experiementan. Recomendamos el siguiente flujo de trabajo para eficientemente reparar tus errores.
+
+1.  _Idenfifica los errores a reparar._ La vista principal en el Dashboard del Reporde de Errores muestra todos los errores ordenados por numero de frecuencia. Buscaras reparar aquellos errores que esten afectando el mayor numero de usuarios. Puedes hacer click a cada error y obtener los detalles como el stack trace , versiones del OS, tipos de dispositivo, y versiones del app afectadas.
+
+2.  _Repara un error._ Haz debugging y repara el error en el codigo cliente.
+
+3.  _Incremente la version de tu build (CFBundleVersion) y compila un nuevo release._
+
+4.  _Marca el error como resuelto y lanza un nuevo build al App Store._ Una vez resuelto, el error no aparecera en la pagina del listado de errores con los filtros por default. si el error ocurre de nuevo en una version posterior, sera automaticamente marcado como no resuelto, y se mostrara en el listado de nuevo. 
+
+5.  Regresa al punto numero 1 y continua mejorando tu app!
+
+## Troubleshooting
+
+### ¿Porque mis crashes no se muestran en el dashboard?
+
+Tu app probablemente no esta encuando los reportes de errores a Parse. Aqui algunos cosas a verificar:
+
+*   Make sure you've enabled Crash Reporting in your App Delegate before you initialize Parse.
```

Comment:
These lines are already translated below, I'm guessing you left these here by mistake.